src/super_gradients/training/datasets/all_collate_functions.py

```diff
@@ -0,0 +1,6 @@
+from super_gradients.training.datasets.datasets_utils import ComposedCollateFunction, MultiScaleCollateFunction
+
+ALL_COLLATE_FUNCTIONS = {
```

Comment:
There are many more collate functions in SG. ctrl+alt+shift+f the project and add them please.

Reply:
added in commit "updated_ fixes"